src/App.js

```diff
@@ -1,25 +1,18 @@
-import logo from "./logo.svg";
-import "./App.css";
+/* eslint-disable import/extensions */
```

Comment:
- [ ] Kindly avoid disabling linters in the project. Linters help developers to write clean code. I there fore suggest to instead fix all the disabled linters.